Sunburst37/ALARM_AND_ABORT.agc

```diff
@@ -94,19 +67,22 @@ JETABORT	TC	ALARM
 
 # CALLING SEQUENCE,
 
-#		TC	ALARM
-#		OCT	AAANN		ALARM NO. NN IN GENERAL AREA AAA.
+		TC	ALARM
+		OCT	AAANN		# ALARM NO. NN IN GENERAL AREA AAA.
 
 #					(RETURNS HERE)
 
+##PAGE 301
```

Comment:
Can you make the styling of these page comments consistent with the page 300 above? Like
```
## Page 301
```